src/api/app/models/issue_tracker.rb

```diff
@@ -32,6 +33,14 @@ def write_to_backend
     self.issues_updated ||= Time.now
   end
 
+  def verify_issue_name!(name)
```

Comment:
Flow control is for what it sounds, control the flow of an algorithm... not to calculate a boolean expression value based on simple conditions, isn't it?
I mean, if I have 2 conditions that returns a simple boolean value I never use a flow control. And if I need to extend (aka add more conditions to it), then I add the new conditions that's all, but if one condition needs some calculations or flow control then is ok, but not in that case from  my POV.

Also for improving the readability you can split it in variables/methods and then do the boolean expression, like:
```ruby
def self.valid_name?(tracker, name)
  is_valid_cve_name = tracker.cve? && Regexp.new(/^\d\d\d\d-\d+$/).match(name)
  is_valid_bugzilla_name = tracker.bugzilla? && name != "exception"
  is_valid_cve_name || is_valid_bugzilla_name
end
``` 

What do you think @hennevogel @bgeuken @ChrisBr?